x/wasm/internal/keeper/reflect_test.go

```diff
@@ -72,14 +72,12 @@ func mustParse(t *testing.T, data []byte, res interface{}) {
 const MaskFeatures = "staking,mask"
 
 func TestMaskReflectContractSend(t *testing.T) {
-	tempDir, err := ioutil.TempDir("", "wasm")
-	require.NoError(t, err)
-	defer os.RemoveAll(tempDir)
-	ctx, keepers := CreateTestInput(t, false, tempDir, MaskFeatures, maskEncoders(MakeTestCodec()), nil)
-	accKeeper, keeper := keepers.AccountKeeper, keepers.WasmKeeper
+	cdc := MakeTestCodec()
```

Comment:
Where do you create and destroy the tmpdir?

Maybe you create it in `CreateTestInput`, but you need to delete it after the tests of we just leave tons of orphan files.